src/js/view/view.js

```diff
@@ -854,8 +854,8 @@ function View(_api, _model) {
     }
 
     function _updateFloating(intersectionRatio) {
-        // Entirely invisible and no floating player already in the DOM.
-        const isVisible = intersectionRatio === 1;
+        // Player is 50% visible or less and no floating player already in the DOM.
+        const isVisible = intersectionRatio > 0.5;
```

Comment:
"viewable" is `intersectionRatio` rounded, so this should be `>= 0.5` or `Math.round(intersectionRatio) === 1`.